### ISSUE TITLE
IP addresses in mixer logs are not readable

### ISSUE THREAD

**Issue by dwradcliffe** [istio/istio#12634]
**Describe the bug**
IP addresses in mixer logs are not readable. (Not Envoy logs)

Examples:
`sourceIp:	 CjQpPQ==`
`destinationIp:	 AAAAAAAAAAAAAP//CjRD8w==`

Istio config resources are defaults as provided by helm charts.
For example this is still present:

```
apiVersion: config.istio.io/v1alpha2
kind: logentry
metadata:
  name: accesslog
  namespace: istio-system
spec:
  variables:
    sourceIp: source.ip | ip("0.0.0.0")
    destinationIp: destination.ip | ip("0.0.0.0")
    <...>
```

**Expected behavior**
I expected to see the string representation of the IP Address.

**Steps to reproduce the bug**
Run any workloads on the mesh. View the mixer access logs.

**Version**
Istio `1.1.0`
Kubernetes `v1.11.6-gke.2`

**Installation**
helm template + (some processing of yaml) + kubectl apply

**Environment**
GKE

**Comment by bianpengyuan** [comment]
cc @gargnupur

**Comment by gargnupur** [comment]
The fix is not there in release 1.1 but we will try to include it in patch release of 1.1
https://github.com/istio/istio/pull/12502

**Comment by dwradcliffe** [comment]
Oh bummer

**Comment by douglas-reid** [comment]
I'm going to close this issue, as the fix is in `master` and `release-1.1` and will be part of the `1.1.2` release, scheduled for release on 4/1/2019.

**Comment by gargnupur** [comment]
@douglas-reid: I think this is not fixed completely.. It is fixed in stackdriver adapter but for the issue reported here, it should be fixed in stdio adapter too... I will work on a fix for that...

**Comment by gargnupur** [comment]
PR in review for fix in stdio adapter: https://github.com/istio/istio/pull/12850#issuecomment-477329496

**Comment by gargnupur** [comment]
Closing as fix is in master and release-1.1 now and will be a part of 1.1.3 release.

**Comment by gargnupur** [comment]
ETA for 1.1.3 is 4/15
